tests/FunctionsTest.php

```diff
@@ -211,19 +211,6 @@ public function testDescribeType()
         $this->assertSame('string(4) "test"', Aws\describe_type("test"));
     }
 
-    /**
-     * @covers Aws\default_http_handler()
-     */
-    public function testGuzzleV6HttpHandler()
```

Comment:
I'd like to keep this, just with the updated namespace/class

Reply:
Done.